pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>org.jboss.aerogear.unifiedpush</groupId>
             <artifactId>unifiedpush-push-model</artifactId>
-            <version>1.1.0-alpha.1</version>
+            <version>1.1.0-alpha.2-SNAPSHOT</version>
```

Comment:
@sebastienblanc the change from 1.1.0-alpha.1 -> 1.1.0-alpha.2-SNAPSHOT is done when using the maven release plugin. Looks like a `git push origin master` was _not_ issued after the -alpha.1 went to maven central :-(